### DISCUSSION TITLE
get version

### DISCUSSION THREAD

**Discussion by Ku100vEvgen** [dioannidis/usbasp#29] [Ideas]
_Features
From version 1.07...
From version 1.08...
From version 1.11..._

but, how i can a read current version from device UsbAsp ?

**Comment by dioannidis** [comment]
Hi @Ku100vEvgen,

in Windows you can find the USBasp's firmware revision using the device manager . In the USBasp device properties, go to the Details tab, select the "Hardware Ids" option and read the numbers in the string starting with "REV_" . In the example photo it says "REV_0111" so the firmware version is 1.11 . Or you can you the USBDeview utility which is more straight forward ...

![USBasp_Version](https://github.com/dioannidis/usbasp/assets/15191633/fd89a400-68dc-491b-b792-0e17885d0ae3)

Hope it helps. 

Regards,
